var/spack/repos/builtin/packages/onednn/package.py

```diff
@@ -125,4 +129,11 @@ def cmake_args(self):
         if self.spec.satisfies("gpu_runtime=ocl"):
             args.append("-DOPENCLROOT=" + self.spec["opencl"].prefix)
 
+        if self.spec.satisfies("+acl"):
+            args.append("-DDNNL_AARCH64_USE_ACL=1")
```

Comment:
```suggestion
        args.append(self.define_from_variant("DNNL_AARCH64_USE_ACL", "acl"))
```
This will also explicitly disable it when `~acl`